osi_object.proto

```diff
@@ -40,6 +40,32 @@ message StationaryObject
     //
     optional string model_reference = 4;
 
+    // Specific information about the classification of stationary objects.
+    //
+    optional StationaryObjectClassification stationary_object_classification = 5;
```

Comment:
@ThomasSchloemicherAVL  All fields should move under "Classification" message that is already available.

Reply:
@kmeids done! Sorry, must have overlooked that.